docs/marketplace/dev/front-end/ui-components-library/data-transformers/collate/filters/equals.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Data Transformer Collate Filter Equals
+description: This document provides details about the Data Transformer Collate Filter Equals service in the Components Library.
+template: concept-topic-template
+---
+
+
+This document provides details about the Data Transformer Collate Filter Equals service in the Components Library.
+
+## Overview
+
+Data Transformer Collate Filter Equals is an Angular Service that implements filtering to equal value of data based on configuration.
+
+Example usage of the Data Transformer Collate Filter Equals in the `@spryker/table` config:
+
+```html
+<spy-table
+  [config]="{
+    datasource: {
+      ...                                               
+      transform: {
+        ...
+        filter: {
+          select1: {
+            type: 'equals',
+            propNames: 'col1',
+          },
+          select2: {
+            type: 'equals',
+            propNames: ['col2', 'col1'],
+          },
+        },
+      },
+    },
+  }"
+></spy-table>
+```
+
+## Interfaces
+
+Below you can find interfaces for the Data Transformer Collate Filter Equals type:
+
+```ts
+interface DataTransformerFilterConfig {
+    type: string;
+    propNames: string | string[];
+}
```

Comment:
Not sure if we need copy-paste this interface for each filter, you already have showed it in the common filters service

Reply:
I think this is necessary because it makes it easier to get information about a filter without going to the main document.